### DISCUSSION TITLE
Got my 5 year old account disabled

### DISCUSSION THREAD

**Discussion by vytskalt** [GooseMod/GooseMod#76] [General]
After switching from Powercord to GooseMod on the browser, for some reason it started saying that my account is likely a spammer. Today it asked me for my phone number and after I verified it, my account got disabled. They sent me an email saying this:
```
Discord has disabled your account for spam and/or platform abuse. Our anti-abuse measures may have flagged your account for any of the following behaviours:
- sending a large number of direct messages in a short span of time
- participating in a server dedicated to spamming Discord
- automating your user account or self-botting
- taking other actions on Discord — for example, joining a ton of servers — faster than humanly possible
- attempting potentially fraudulent activity
```
I haven't done anything in that list.
Could this be caused by GooseMod? I rememeber downloading some of the most popular plugins, and that's it.
I've also recently changed my accounts email to simplelogin, maybe that affected something.

**Comment by CanadaHonk** [comment]
Nothing in GM uses the Discord API so it cannot cause this. Marking as a likely spammer probably means you were (falsely) reported by someone.

**Reply by vytskalt** [reply]
Could a plugin have caused this?

**Reply by CanadaHonk** [reply]
No since no plugins use the Discord API. It was probably your email change or someone else reporting you, no one else (out of ~100k users) has reported being disabled for GM so highly doubt it is because of it.
